logging/google/cloud/logging/handlers/handlers.py

```diff
@@ -90,7 +96,7 @@ def emit(self, record):
         :param record: The record to be logged.
         """
         message = super(CloudLoggingHandler, self).format(record)
-        self.transport.send(record, message)
+        self.transport.send(record, message, self.resource)
```

Comment:
Nit: when an argument is a keyword / optional argument, always specify it by name: `resource=self.resource`.